docs/configuration.md

```diff
@@ -54,3 +54,40 @@ Then those packages are usable directly:
    plt.plot(np.sin(np.linspace(0, 20, 100)))
    plt.show();
 ```
+
+## Advanced Configuration
+
+```{warning}
+This section is mostly for reference and should not be needed for regular use of the `jupyterlite-xeus-python` kernel.
+```
+
+### Provide a custom `empack_config.yaml`
+
```

Comment:
Maybe add a paragraph first saying why you would nee to provide a custom config.

```md
Packages sometimes ship more data than needed for the package to work (tests, documentation, data files etc). This is fine on a regular installation of the package, but in the emscripten case this means that starting the kernel would download all those unused files.
For this reason, empack would filter out anything that is not required for the Python code to run. It does it by following a set of filtering rules.
But this default filtering could break some package, in that case you would probably want to either contribute to the default empack config, or provide your own set of filtering rules. 
```

Reply:
Sounds good. It is mentioned as a regular sentence a bit below but it would be better to make it a `note` at the top of the section.